### PR TITLE
Support for Phi3 Vision Model

### PR DESCRIPTION
**Pull request by baijumeswani**

### CODE REVIEW

File: examples/python/phi3v.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    return (glob.glob(text+'*')+[None])[state]
+
+def run(args: argparse.Namespace):
+    print("Loading model...")
+    model = og.Model(args.model_path)
+    processor = model.create_multimodal_processor()
+    tokenizer_stream = processor.create_stream()
+
+    while True:
+        # image_path = input("Image Path (leave empty if no image): ")
```
Comment (kunal-vaishnavi) [inline examples/python/phi3v.py:21]:
```suggestion
```

File: examples/c/src/phi3v.cpp [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+int main(int argc, char** argv) {
+  if (argc != 2) {
+    print_usage(argc, argv);
+    return -1;
+  }
+
+  std::cout << "--------------------" << std::endl;
+  std::cout << "Hello, Phi-3-Mini-V!" << std::endl;
```
Comment (kunal-vaishnavi) [inline examples/c/src/phi3v.cpp:97]:
```suggestion
  std::cout << "Hello, Phi-3 Vision!" << std::endl;
```

File: examples/python/phi3v.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            new_token = generator.get_next_tokens()[0]
+            print(tokenizer_stream.decode(new_token), end='', flush=True)
+        
+        for _ in range(3):
+            print()
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--model_path", type=str, required=True, help="Path to the model")
```
Comment (natke) [inline examples/python/phi3v.py:62]: Can we add "-m" here too? (For the lazy people, like me!)

**Review by yufenglee — APPROVED** [review]

File: examples/python/phi3v.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            new_token = generator.get_next_tokens()[0]
+            print(tokenizer_stream.decode(new_token), end='', flush=True)
+        
+        for _ in range(3):
+            print()
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-m", "--model_path", type=str, required=True, help="Path to the model")
```
Comment (yufenglee) [inline examples/python/phi3v.py:62]: we may improve the example by adding more options to control logging level and performance(time to process image, token generation speed)

**Pull request merged**